.eslintrc.js

```diff
@@ -100,7 +100,6 @@ module.exports = {
         {
           'files': [
             'playground/**/authenticator-verification-okta-verify-reject-push.json',
-            'playground/**/error-with-failure-redirect.json',
```

Comment:
Might get some merge conflict as I made a recent change to clean up this list https://github.com/okta/okta-signin-widget/pull/1999